### ISSUE TITLE
No alternative text for download button within viewer

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#1536]
![screen shot 2017-01-24 at 15 19 07](https://cloud.githubusercontent.com/assets/13200338/22253259/d3ce35b0-e248-11e6-8a0c-eaac0b21f716.png)
![screen shot 2017-01-24 at 15 18 21](https://cloud.githubusercontent.com/assets/13200338/22253260/d3e3fce2-e248-11e6-91d4-fdaf3b072da8.png)
